### PR TITLE
extract functions from bitcoinjs-lib

### PR DESCRIPTION
**Pull request by fanatid**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,42 @@
 {
   "name": "crypto-hashing",
-  "version": "0.3.1",
-  "description": "Easy interfaces for cryptographic hash functions",
+  "version": "1.0.0",
```
Comment (jprichardson) [inline package.json]: Please squash this. Version bumps should be in there own commit.

Comment (fanatid) [inline package.json]: Done.

**Comment by fanatid** [comment]
as alternative we can reexport `createHash` with additional `hash160`/`hash256`:
```js
var cryptoHashing = require('crypto-hashing')
cryptoHashing('sha256', buffer) // give 32-bytes buffer
cryptoHashing('ripemd160', buffer) // 20-bytes buffer
cryptoHashing('hash256', buffer) // 32-bytes buffer
// and so on
```

it gives more available hashes from `createHash` and less code

**Comment by jprichardson** [comment]
> as alternative we can reexport createHash with additional hash160/hash256

IDC. What do you think is best?

**Comment by fanatid** [comment]
> > as alternative we can reexport createHash with additional hash160/hash256

> IDC. What do you think is best?

I'm already changed :)
main function:
```js
module.exports = function cryptoHash (algorithm, buffer, encoding) {
  if (algorithm === 'hash160') return hash160(buffer, encoding)
  if (algorithm === 'hash256') return hash256(buffer, encoding)
  return createHash(algorithm).update(buffer).digest(encoding)
}
```
